### PR TITLE
New: (Indexer) Torrent Ncore

### PR DESCRIPTION
**Pull request by laszlodoczi**
- Added yaml based Ncore tracker

Closes https://requests.prowlarr.com/b/feedback/p/torrent-ncore

### CODE REVIEW

File: definitions/v2/ncore.yml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+    download:
+      selector: div[class^="torrent_txt"] > a
+      attribute: href
+      filters:
+        - name: querystring
+          args: id
+        - name: prepend
+          args: "ajax.php?action=torrent_drop&id="
+
```
Comment (bakerboy448) [inline definitions/v2/ncore.yml:140]: Nit no new line

File: definitions/v2/ncore.yml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  fields:
+    category:
+      selector: .box_alap_img > a[href^="/torrents.php?tipus="]
+      attribute: href
+      filters:
+        - name: querystring
+          args: tipus
+        - name: trim
+
```
Comment (bakerboy448) [inline definitions/v2/ncore.yml:118]: No line breaks between fields

File: definitions/v2/ncore.yml [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    title:
+      selector: div[class^="torrent_txt"] > a
+      attribute: title
+
```
Comment (bakerboy448) [inline definitions/v2/ncore.yml:122]: Same as above

File: definitions/v2/ncore.yml [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    details:
+      selector: div[class^="torrent_txt"] > a
+      attribute: href
+      filters:
+        - name: querystring
+          args: id
+        - name: prepend
+          args: "torrents.php?action=details&id="
+
```
Comment (bakerboy448) [inline definitions/v2/ncore.yml:131]: Same as above

File: definitions/v2/ncore.yml [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+        - name: prepend
+          args: "ajax.php?action=torrent_drop&id="
+
+    poster:
+      selector: img.infobar_ico[onmouseover]
+      attribute: onmouseover
+      filters:
+        - name: regexp
+          args: "'(.+?)'"
+
```
Comment (bakerboy448) [inline definitions/v2/ncore.yml:147]: Same as above

**Review by bakerboy448 — CHANGES_REQUESTED** [review]
See comments from initial review.

Was this tested on a old Prowlarr running Cardigann v2? It appears it would not work with a Prowlarr running Cardigann v2.

Based on the fact description is missing and thus optional, this would at minimum need to v3.   Haven't looked in depth enough to see if it uses any Cardigann v4 or v5 features

- address comments
- move to the correct Prowlarr Cardigann version

**Comment by laszlodoczi** [comment]
> See comments from initial review.
> 
> Was this tested on a old Prowlarr running Cardigann v2? It appears it would not work with a Prowlarr running Cardigann v2.
> 
> Based on the fact description is missing and thus optional, this would at minimum need to v3. Haven't looked in depth enough to see if it uses any Cardigann v4 or v5 features
> 
>     * address comments
> 
>     * move to the correct Prowlarr Cardigann version

I think I used a v2 as a base.
Moved the file into v5, so there should be no issues with versions. Is this fine?

**Comment by bakerboy448** [comment]
v2 _may_ be fine

v3 should be the safest bet, so swap that real quick and we'll be all set.

and v5 is only in nightly at this point and this one doesn't seem to have any v5 only features ;)

**Comment by laszlodoczi** [comment]
@bakerboy448 done!

**Comment by Qstick** [comment]
Seems fine here

**Review by bakerboy448 — APPROVED** [review]

**Comment by bohemtucsok** [comment]
Hi 

Ncore torrent problem 

Query successful, but no results were returned from your indexer. This may be an issue with the indexer or your indexer category settings.

anyone have any ideas?

**Comment by bakerboy448** [comment]
Please use one of the support channels: [Reddit](https://www.reddit.com/r/prowlarr/) or  [Discord](https://prowlarr.com/discord) for support/questions.

https://wiki.servarr.com/prowlarr/faq#prowlarr-will-not-sync-x-indexer-to-app

**Pull request merged**